pages/linux/lsusb.md

```diff
@@ -0,0 +1,23 @@
+# lsusb
+
+> Display info about connected USB devices.
+
+- List all the USB ports available:
+
+`lsusb`
+
+- Hierarchy view, with each device's supported speed:
+
+`lsusb -t`
+
+- Get detailed info of a USB device:
+
+`lsusb -D {{device}}`
+
+- Show only devices with the specified vendor and product ID:
```

Comment:
Again just to maintain the coherence of the page,
`List devices with a specified vendor and product id only`